src/kontrol/deployment.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from collections.abc import Iterable
+
+
+class DeploymentSummary:
+    SOLIDITY_VERSION = '^0.8.13'
+
+    name: str
+    commands: list[str]
+    accounts: dict[str, str]  # address, name
+    code: dict[str, str]
+
+    def __init__(self, name: str, accounts: dict | None = None) -> None:
+        self.commands = []
+        self.accounts = accounts if accounts is not None else {}
+        self.name = name
+        self.code = {}
+        for acc_key in list(self.accounts):
+            self.accounts[acc_key] = self.accounts[acc_key]
+
+    def generate_header(self) -> list[str]:
+        lines = []
+        lines.append(f'pragma solidity {self.SOLIDITY_VERSION};\n')
+        lines.append('import { Vm } from "forge-std/Vm.sol";\n')
+        return lines
+
+    def generate_code_contract(self) -> list[str]:
+        lines = []
+        lines.append(f'contract {self.name}Code ' + '{')
+        for code_alias, code in self.code.items():
+            lines.append(f'\tbytes constant public {code_alias}Code = hex{code!r};')
```

Comment:
```suggestion
            lines.append(f'\tbytes constant internal {code_alias}Code = hex{code!r};')
```
No need to create functions for the code

Reply:
Fixed.